src/main/java/io/strimzi/kafka/bridge/http/HttpBridge.java

```diff
@@ -260,6 +261,10 @@ private void seekToEnd(RoutingContext routingContext) {
         processConsumer(routingContext);
     }
 
+    private void readinessCheck(RoutingContext routingContext) {
+        routingContext.response().setStatusCode(200).end();
```

Comment:
Now we have an `HttpUtils` class for sending response to the HTTP clients, we have to use that one. And I am using `HttpResponseStatus` class for the status codes (i.e. `OK` which is 200)